cranelift/codegen/src/ir/instructions.rs

```diff
@@ -797,15 +797,22 @@ impl OperandConstraint {
                 tys.lanes = BitSet::from_range(0, 1);
 
                 if ctrl_type.is_int() {
-                    // The interval should include all types wider than `ctrl_type`, so we use
-                    // `2^8` as the upper bound, and add one to the bits of `ctrl_type` to define
-                    // the interval `(ctrl_type, I128]`.
-                    tys.ints = BitSet::from_range(ctrl_type_bits as u8 + 1, 8);
+                    let lower_bound = ctrl_type_bits as u8 + 1;
+                    // The largest integer type we can represent in `BitSet8` is I128, which is
+                    // represented by bit 7 in the bit set. Adding one to exclude I128 from the
+                    // lower bound would overflow as 2^8 doesn't fit in a u8, but this would
+                    // already describe the empty set so instead we skip initializing it at all.
```

Comment:
"skip initializing" is somewhat ambiguous here; makes it sound like we're either leaving memory uninitialized or passing through an existing value or something; can we say "so we leave `ints` at its default empty state" or something instead?

Reply:
Good point, I'll change this.